packages/devtools_app/lib/src/screens/inspector/inspector_data_models.dart

```diff
@@ -680,28 +695,31 @@ class FlexLayoutProperties extends LayoutProperties {
   }
 
   List<RenderProperties> crossAxisSpaces({
-    @required List<RenderProperties> childrenRenderProperties,
-    @required double Function(Axis) maxSizeAvailable,
+    required List<RenderProperties>? childrenRenderProperties,
+    required double Function(Axis)? maxSizeAvailable,
   }) {
     if (crossAxisAlignment == CrossAxisAlignment.stretch) return [];
     final spaces = <RenderProperties>[];
     for (var i = 0; i < children.length; ++i) {
       if (dimension(crossAxisDirection) ==
               displayChildren[i].dimension(crossAxisDirection) ||
-          childrenRenderProperties[i].crossAxisDimension ==
-              maxSizeAvailable(crossAxisDirection)) continue;
+          childrenRenderProperties![i].crossAxisDimension ==
+              maxSizeAvailable!(crossAxisDirection)) continue;
 
       final renderProperties = childrenRenderProperties[i];
       final space = renderProperties.clone()..isFreeSpace = true;
 
       space.crossAxisRealDimension =
-          crossAxisDimension - space.crossAxisRealDimension;
+          crossAxisDimension! - space.crossAxisRealDimension!;
       space.crossAxisDimension =
           maxSizeAvailable(crossAxisDirection) - space.crossAxisDimension;
       if (space.crossAxisDimension <= 0.0) continue;
       if (crossAxisAlignment == CrossAxisAlignment.center) {
         space.crossAxisDimension *= 0.5;
-        space.crossAxisRealDimension *= 0.5;
+        final crossAxisRealDimension = space.crossAxisRealDimension;
+        if (crossAxisRealDimension != null) {
+          space.crossAxisRealDimension = crossAxisRealDimension * 0.5;
+        }
```

Comment:
this can be reverted once height and width are made non nullable

Reply:
done